packages/ui/src/components/NumberInput/NumberInput.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import {
+  forwardRef,
+  HTMLChakraProps,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput as ChakraNumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  UseNumberInputProps,
+} from '@chakra-ui/react';
+
+type NumberInputSize = 'sm' | 'md' | 'lg';
+
+type NumberInputMode = 'numeric' | 'decimal';
+
+export interface NumberInputProps extends Omit<HTMLChakraProps<'div'>, keyof UseNumberInputProps> {
+  /**
+   * Determines number input's paddings and height.
+   */
+  size: NumberInputSize;
+  /**
+   * Determines type of data that might be entered by the user (numeric or decimal).
+   */
+  inputMode?: NumberInputMode;
+  /**
+   * Determines if number input is disabled or not .
+   */
+  disabled?: boolean;
+  value?: number;
+  onChange?: (arg: string) => void;
```

Comment:
Correct type for this function is `(stringValue: string, numberValue: number) => void`